mysite/settings/tests.py

```diff
@@ -0,0 +1,20 @@
+from .base import *  # noqa
```

Comment:
Why do `mysite/settings/testing.py` and `mysite/settings/tests.py` (later `ynr/settings/testing.py` and `ynr/settings/tests.py`) need to be 2 different files?